src/asmcnc/apps/wifi_app/screen_wifi.py

```diff
@@ -592,19 +674,28 @@ def update_strings(self):
         self.password_label.text = self.l.get_bold("Password")
         self.country_label.text = self.l.get_bold("Country")
         self.connect_button.text = self.l.get_str("Connect")
+        self.custom_ssid_input()
+        self.custom_network_name.hint_text = self.l.get_str("Enter network name")

```

Comment:
What happens when this does not exist? Is that why the text doesn't change language I wonder 🤔